python-project-template/{% if include_docs %}docs{% endif %}/notebooks/README.md

```diff
@@ -10,4 +10,13 @@ This means they should be able to be run with the resources in the repo, and in
 
 This is great for notebooks that can run in a few minutes, on smaller datasets.
 
+If you would like to include these notebooks in automatically generated documentation
+simply add the notebook name to the ``../notebooks.rst`` file. Be aware that you may
```

Comment:
I always do it wrong because I forget to add `#Title` markdown cell to the notebook. Should we mention it here and/or docs?

Reply:
Yep, I'll add a line about that!

Reply:
Done